# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1292711579 18000
#      Sat Dec 18 17:32:59 2010 -0500
# Node ID 98d523608f70f583bf478050e3a3fe2027f02554
# Parent  7ca273af4309616306c9d6fbecec436339e160b2
view.m: Return azimuth and elevation for nargin == 0.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-12-18  Ben Abbott  <bpabbott@mac.com>
+
+	* plot/view.m: Return azimuth and elevation for nargin == 0.
+
 2010-10-28  Rik  <octave@nomad.inbox5.com>
 
 	* deprecated/module.mk, sparse/module.mk, deprecated/sphcat.m,
 	deprecated/spvcat.m: Deprecate sphcat and spvcat functions.
 
 2010-12-17  John W. Eaton  <jwe@octave.org>
 
 	* plot/comet3.m: New function.
diff --git a/scripts/plot/view.m b/scripts/plot/view.m
--- a/scripts/plot/view.m
+++ b/scripts/plot/view.m
@@ -37,19 +37,17 @@
 ## @end deftypefn
 
 ## Author: jwe
 
 function [azimuth, elevation] = view (varargin)
 
   if (nargin < 4)
     if (nargin == 0)
-      tmp = get (gca (), "view");
-      az = tmp(1);
-      el = tmp(2);
+      args = {get(gca (), "view")};
     else
       ax = varargin{1};
       if (ishandle (ax) && strcmp (get (ax, "type"), "axes"))
         args = varargin(2:end);
       else
         ax = gca;
         args = varargin;
       endif
